client/my-sites/patterns/components/get-started/index.tsx

```diff
@@ -70,7 +74,11 @@ export function PatternsGetStarted() {
 					<img
 						className="patterns-get-started__item-image"
 						src={ imagePageLayouts }
-						alt=""
+						alt={ translate( "A support page with the title 'Understand Page Layouts'", {
```

Comment:
I'm really splitting hairs here but that's just to be consistent with the picture:
```suggestion
						alt={ translate( "Support page with the title 'Understand page layouts'", {
```